Documentation/Administration/Docker/AutomateSetup/_codesnippets/_Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM martinhelmich/typo3:13.4
+
+USER root
```

Comment:
You are switching to root here, it would make sense to switch back to `www-data` to make the container run rootless. 

This removes the need for `gosu`

Reply:
I think as this chapter logically builds on the previous chapter, where we introduced entrypoints I would leave it as is